maths/makevalid/makevalid_test.go

```diff
@@ -10,8 +10,16 @@ import (
 	"github.com/go-test/deep"
 	"github.com/terranodo/tegola/maths"
 	"github.com/terranodo/tegola/maths/hitmap"
+	"github.com/terranodo/tegola/maths/points"
 )
 
+const TileBuffer = 16
+
+var extent = points.Extent{
+	{0.0 - TileBuffer, 0.0 - TileBuffer},
+	{4096.0, 4096.0},
```

Comment:
should this have TileBuffer added as well?

Reply:
Yes, it should.